codegen/docs/example_graph.drawio

```diff
@@ -0,0 +1,2 @@
+<mxfile host="Electron" modified="2021-02-15T22:55:27.277Z" agent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/12.6.5 Chrome/80.0.3987.86 Electron/8.0.0 Safari/537.36" etag="Nrx1mxvLROi_wSdW3rk9" version="12.6.5" type="device" pages="2"><diagram id="GDvEBKuDxmKS5widdRZa" name="original">5ZhRb5swEMc/DY+dwG6y9DFN6TZpkaal0pZHK9zAHWBkjgD79DPBYBgjSrUmZOpTuD/n43z3OwhYdBUVHyRLgrXwILSI7RUWfbAIcRybqJ9KKbVi23at+JJ7WjPChv+CxlGrGfcg7TmiECHypC/uRBzDDnsak1LkfbcfIuxfNWE+DITNjoVD9Rv3MKjVxcw2+kfgfoDtjvWZiDXOWkgD5om8I1HXoispBNZHUbGCsKpeU5d63ePI2TYxCTGesiBKt0t35d48/0z34Dw93Ufrrze3Ojcsmw2Dp/avTSExEL6IWega9V6KLPagimory/h8FiJRoqPEZ0AsdTNZhkJJAUahPqsSluX3av27WWNudbiD8VD0rFJbda5VgqMl0FIqMrmDI/tuUGLSBzziR9pGKcRBRKDyUeskhAz5vp8H06j5rZ/phjrQDXlBc3TcPQszfaWl5w0a1m9HHnCETcIOu8/VUPZLryOCRCiOF3G4ab2ANITrkW4BzzvzoaWgMxrUPlOZZm+UYXIiw3RKhsmA4XUWXh/Di4kZpoMyrUS8J9c37fR24kq9f6PTPj9x2kfaeZlpnw8w/hQnGSrJGbRtapKd+cQk301Lco9jg/UIyVBw7AyAsrZNRHVsFlXGGehf/A/0L/52E09xcvKdayPfGd4NLom+8zL0XxHj5q327O8dh6VLKVnZcUgEjzHtRP5SCYaT9tne/Cu6m/3R6jqiaXyb2j+wYI8+M8jkk0MvNznKNN8C6uKaTyrU/Q0=</diagram><diagram id="42WywKGboth1uqFjvlTy" name="compiled">1VdNc5swEP01HNvhw1D3GGM3aZPMNONDklNGRTLQAiJiwZBfXymSwDJOk7hN7F5s7ZN2kd6+Z2HLC/P2lKEyuaSYZJZr49by5pbrBp7NPwXQScD/5EggZimW0AawTB+IAlVeXKeYVMZCoDSDtDTBiBYFicDAEGN0bS5b0cx8aoliMgKWEcrG6HWKIZHo1LcH/IykcaKf7NhqJkd6sQKqBGG63oC8heWFjFKQo7wNSSa407xUOAxYd3Nx7q/msxznEF/dfJDFvrwmpT8CIwXsXfr016K4ppfs6mF6bt831Ulz9k2XblBWK77CugKaW26Q8UfNfjA+isWIf9/X4qizmBSEISD4blUX0Z09zCieoNPkM1oXmIgN2Hx6naRAliWKxOyaq41jCeQZj5w+uyEMSLvVu2cO7vTd4ComNCfAOp6nq0x9WUcpuG/oetCDo+WabGhBSx8pCcZ96YFmPlBMv4J1d0QUwVy0KqQMEhrTAmWLAZ2ZVA5rLigtFYE/CUCnHIhqoCa9nEJOnMj/6OvwVpV7DOatEXU6alPYSOPRra7Ix0OSCHSOPJ841B6N5MTQmkXkedkCYjGBP6yb7BYGIxmCtDE3t6vLKvU7Tfm2e0G5E9sQlOdsCUXuX2VtaaXfxv7ymYxNS4vGxcdpP9c22XKnY/tN39N9/v/kvrd2UvBCJz3R4xc76a86FowE/7Uoa+FIZ9TLo9S8ExxY858PqnnHUPxggANpXv9cv7noH1NPGEPdxoJSXAvV07eLt3W7uJ6/1X5Z8Z9eKZqSHRYbv6wchcW897MYD4e3fMn48FfJW/wG</diagram></mxfile>
```

Comment:
this file starts with a space. if this is a geneted file, let's go as-is, if not could you please remove the first space and empty line at the end?

Reply:
Yes, this is generated file, but format checker failed due absent newline at the end before.
I added new line in the end manually.

I can try to add exception in checker, if it possible.

Reply:
> I added new line in the end manually. I can try to add exception in checker, if it possible.

OK, as this seems low priority, let's do this later :)